compiler/rustc_error_codes/src/error_codes/E0461.md

```diff
@@ -0,0 +1,30 @@
+Couldn't find crate `..` with expected target triple `..`.
+
+Consider the following two files:
```

Comment:
Please follow the format and use instead "Example of erroneous code".

Reply:
Done. IMO, [RFC 1567](https://rust-lang.github.io/rfcs/1567-long-error-codes-explanation-normalization.html) is a bit outdated. It doesn't address these kinds of multi-file errors and other atypical errors. Also this really should be linted.